docs/subnet.md

```diff
@@ -31,8 +36,12 @@ $ docker exec -it ipc_root_1234 eudico wallet export --lotus-json t1cp4q4lqsdhob
 ```
 
 ### Importing wallet keys
+Your agent handles the keys for all of your addresses in IPC and is responsible for signing the transactions to the different networks. To import a key to the agent you need can use: 
+```bash
+`./ipc-agent/bin/ipc-agent wallet import --path=<wallet-key-file-path>`
+```
 
-Depending on whether the subnet is running inside a docker container or not, you may need to import keys into a node. You may use the following commands to import a wallet to a subnet node: 
+Depending on whether the subnet is running inside a docker container or not, you may need to import keys directly into a node to run your validator. You may use the following commands to import a wallet to a subnet node: 
```

Comment:
Could you ellaborate? I understand that the export methodology differs based on circumstances, but it's less obvious to me when you'd need to import into the node vs. agent.

Reply:
Fixed.